src/group_theory/subgroup.lean

```diff
@@ -1991,6 +2017,26 @@ begin
     rw [inf_of_le_right (sup_le hA hA'), inf_of_le_right hA', inf_of_le_right hA] },
 end
 
+@[to_additive] lemma normal_subgroup_mul
```

Comment:
I'm struggling here - is this different to the lemma of the same name in the other file?

Reply:
Oops, this sneaked in as a duplicate.